src/mbgl/map/source.cpp

```diff
@@ -39,9 +39,7 @@ Source::Source() {}
 Source::~Source() = default;
 
 bool Source::isLoaded() const {
-    if (!loaded) {
```

Comment:
Doesn't `req` imply `!loaded` though? `loaded` only gets set to true after all the requests are successful.